### ISSUE TITLE
jpeg.port: update sha hash

### ISSUE THREAD

**Issue by m-stein** [genodelabs/genode#4406]
I don't know why (and indeed it seems suspicious to me) but the SHA of the JPEG download doesn't work anymore for me (genodelabs/staging). I've created a simple fix but am unsure whether we should investigate the cause of the regression.

**Comment by m-stein** [comment]
07f12671a1 jpeg.port: update sha hash

**Comment by chelmuth** [comment]
I checked the contents by

```
tar --one-top-level --strip-components=1 -xf jpegsrc-old.v9d.tar.gz
tar --one-top-level --strip-components=1 -xf jpegsrc-new.v9d.tar.gz
diff -ur jpegsrc-old.v9d jpegsrc-new.v9d

tar -tvf  jpegsrc.v9d-old.tar.gz | sort -k6 > old.lst
tar -tvf  jpegsrc.v9d-new.tar.gz | sort -k6 > new.lst
diff -u old.lst new.lst
```

and merged the slightly adjusted f928570a625ca1c18782334ea406c3daad1ca0b6.
